tests/ts_cluster_keys.erl

```diff
@@ -0,0 +1,169 @@
+-module(ts_cluster_keys).
```

Comment:
Sorry to be nit picky here, @andytill, but how about `ts_cluster_flexible_keys`? Just `keys` seems a wee bit vague.